src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -270,9 +275,11 @@ def initialize_newborns(self, pop_data: SimulantData) -> None:
         new_births = new_births.merge(
             mothers[inherited_traits], left_on="parent_id", right_index=True
         )
-        new_births["relation_to_household_head"] = new_births[
-            "relation_to_household_head"
-        ].map(metadata.NEWBORNS_RELATION_TO_HOUSEHOLD_HEAD_MAP)
+        new_births["relation_to_household_head"] = (
+            new_births["relation_to_household_head"]
+            .map(metadata.NEWBORNS_RELATION_TO_HOUSEHOLD_HEAD_MAP)
+            .astype(new_births["relation_to_household_head"].dtype)
```

Comment:
I don't know if this would be bad form to do in a metadata file, but you could consider defining the dtypes themselves there in addition to the lists of values.

Reply:
I need to think a bit more about where a lot of these constants should live in general.

Reply:
Maybe for now you could do `.astype(pd.CategoricalDtype(categories=metadata.RELATIONSHIPS))` here? But, it's also okay how it is. Just took me a second to parse.

Reply:
We should definitely have a centralized schema approach. One option @rmudambi is to rip off what we were doing on the Covid ETL. I can look in more detail but we basically had a Schema class and you could specify dataset required columns (including their dtypes, nullability, etc). Something like:
```
output_schema = Schema(
    RequiredColumn("address_id", dtype=int, nullable=False),
```

And then there were soft and hard `check_constraint` methods attached to that Schema class